### DISCUSSION TITLE
Confused about the header being added to NES dumps in V9.1

### DISCUSSION THREAD

**Discussion by Rot-gut** [sanni/cartreader#453] [Need help]
I updated to V9.1 for my Open Source Cartridge Reader. When I dump a NES game, the CRC32 is calculated correctly and a renamed .NES file is outputted (although TBH I was hoping the headerless .BIN could be outputted, too).

When I compare the checksum of the .NES file to NES 2.0 headers, the headers are different resulting in a different checksum. What kind of headers are being added to these dumps? I don't fully understand the source. It claims No-Intro, but No-Intro is not using the headers that the cart reader is adding.

Example: Adventures in the Magic Kingdom for NES (USA)
- The header given by the cart reader for the .NES file is 4E45531A081010080000070700000001
- The header given by No-Intro and nes.dnsabr are different from what the cart reader gives.
- 
- No-Intro: 4E45531A081010080000000000000001
- Source: https://datomatic.no-intro.org/index.php?page=show_record&s=45&n=0050
- 
- nes.dnsabr: 4E45531A081010000000000000000000
- Source: http://nes.dnsabr.com/entity/view?id=50

**Comment by sanni** [comment]
Try updating to V9.4 (both Arduino and SD files): https://github.com/sanni/cartreader/archive/refs/heads/master.zip
That should fix the differences in the iNES header and give you an option to dump the bin without iNES header in the PRG/CHR submenu.

The CRC is calculated excluding the iNES header since the header is just a made-up thing and not part of the actual cartridge data.

**Reply by Rot-gut** [reply]
Thank you. I don't see arduino.exe within this master zip. Can you please let me know how it should be installed?

**Reply by Rot-gut** [reply]
Ah nevermind, found it here! :) https://github.com/sanni/cartreader/releases/tag/V9.4
